models/vision/classification/alexnet/run_alexnet.sh

```diff
@@ -16,8 +17,15 @@ $curr_dir/../../get_images.sh $image_dir
 if [[ $TEST_WITH_GPU -eq 1 ]]; then
   echo "======== Testing with ODLA TensorRT ========"
   python3 $curr_dir/../../invoke_halo.py --model $model_file --label-file $curr_dir/../1000_labels.txt --image-dir $image_dir --odla tensorrt
+# RUN: FileCheck --input-file /tmp/alexnet_tensorrt.txt %s
 fi
 
 # Using HALO to compile and run inference with ODLA XNNPACK
 echo "======== Testing with ODLA DNNL ========"
-python3 $curr_dir/../../invoke_halo.py --model $model_file --label-file $curr_dir/../1000_labels.txt --image-dir $image_dir --odla dnnl
\ No newline at end of file
+python3 $curr_dir/../../invoke_halo.py --model $model_file --label-file $curr_dir/../1000_labels.txt --image-dir $image_dir --odla dnnl
+# RUN: FileCheck --input-file /tmp/alexnet_dnnl.txt %s
+
+# CHECK: /tmp/images/dog.jpg ==> "wallaby, brush kangaroo",
```

Comment:
Can we do "CHECK: dog.jpg ==> "...." 

Matching for "/tmp/images" might cause issues if the temporary dir name is changed in future.

Reply:
yes, you are right